### DISCUSSION TITLE
Getting width/height of a node

### DISCUSSION THREAD

**Discussion by CodyBontecou** [bcakmakoglu/vue-flow#1384] [Q&A]
`toObject()` doesn't return width or height values that are set using `NodeResizer`. 

I'm able to see the height/width using the `@resize` emit and figure I can manage it using that.
```ts
    <NodeResizer
      :min-width="100"
      :min-height="30"
      @resize="e => console.log(e)"
    />
```

 but I'm curious if there's an easier way to manage this information.

**Comment by bcakmakoglu** [comment]
Mh... good point, internal values get omitted when calling `toObject` but it might make sense to keep dimensions in there 🤔

**Reply by CodyBontecou** [reply]
It would be helpful to have that option! I'm attempting to persist nodes but it loses the dimensions that are set using the NodeResizer component.

**Reply by sefgit** [reply]
badly needed feature to consistently restore saved nodes.

**Reply by plc-dev** [reply]
+1 

As mentioned here: #1627 the OnResize-Events could have a similar interface to the NodeDragEvent.
I could add a PR if required @bcakmakoglu.
